core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/ApiServiceSpec.scala

```diff
@@ -290,7 +294,6 @@ trait ApiServiceSpec
         billingProfileManagerDAO,
         workspaceManagerDAO,
         executionServiceCluster.readMembers.map(c => c.key -> c.dao).toMap,
-        Seq("my-favorite-group"),
```

Comment:
How come you needed to drop this line?

Reply:
correct, this was the list of groups to check and we are not checking groups anymore